packages/vuepress-theme-odyssey/components/DocsNav.vue

```diff
@@ -88,8 +88,12 @@ export default {
       type: Array, // TODO: figure out if theres a way to better 'type' this with vue
       required: true
     },
+    forceExpand: {
+      type: Boolean,
+      default: false
+    },
```

Comment:
Shouldn't `isSubmenuExpanded` already handle this?

Reply:
`isSubmenuExpanded` deals with whether or not the submenu is visible (or expanded)

Perhaps a clearer name would be `isSubmenuVisible`? Im open to suggestions here, its not as clear as I would like.